composer.json

```diff
@@ -255,6 +255,7 @@
         "spryker/scheduler-jenkins": "^1.2.1",
         "spryker/secrets-manager": "^1.0.0",
         "spryker/secrets-manager-aws": "^1.0.1",
+        "spryker/security-blocker-merchant-portal-gui": "^1.0.0",
```

Comment:
It seems like `spryker-shop/security-blocker-page` should also be updated according to https://github.com/spryker/spryker-shop/pull/2332